### PR TITLE
Feature/tiny 6866

### PR DESCRIPTION
**Pull request by techtangents**

### CODE REVIEW

File: src/test/ts/commands/PublishTest.ts [hunk 1]
```diff
@@ -1,9 +1,12 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```
Comment (ashfordneil) [inline src/test/ts/commands/PublishTest.ts:1]: I'd very much prefer that as a disable-next-line thing, or removed if we can do it

**Review by ashfordneil — APPROVED** [review]

File: src/test/ts/commands/PublishTest.ts (hunk 1 above)
Comment (techtangents) [reply src/test/ts/commands/PublishTest.ts]: Thanks. Yeah, it's not needed.

Comment (techtangents) [reply src/test/ts/commands/PublishTest.ts]: Fixed.

**Review by JamesToohey — COMMENTED** [review]
Should we add instructions for this argument option to the README? I also noticed the other arguments aren't really in there either

**Review by lnewson — APPROVED** [review]

**Comment by techtangents** [comment]
> Should we add instructions for this argument option to the README? I also noticed the other arguments aren't really in there either

I worry that this would just duplicate the `--help` text and we'd have to keep them in sync. Instead, I've added a comment in the readme, referring to `--help`.

**Pull request merged**